macro/macro.go

```diff
@@ -101,14 +107,16 @@ func (m *macro) compile(input string) error {
 				})
 			}
 			currentToken.Reset()
-			ismacro = true
+			isMacro = true
 			i++
 			continue
 		}
-		if ismacro {
+
+		if isMacro {
 			if c == '}' {
 				// we close a macro
-				ismacro = false
+				isMacro = false
+				// TODO(jcchavezs): can key be empty? e.g. %{var}
```

Comment:
Ping @airween @fzipi @jptosso

Reply:
We should test in modsec

Reply:
Syntax error unless it's a single variable like request_uri